lib/ansible/plugins/loader.py

```diff
@@ -129,6 +129,20 @@ def __init__(self):
         self.removal_version = None
         self.deprecation_warnings = []
         self.resolved = False
+        self.plugin_type = None
+
+    @property
```

Comment:
Vestigial?

Reply:
Thanks. Too many iterations of this thing :-)